beacon-chain/rpc/prysm/v1alpha1/validator/proposer_test.go

```diff
@@ -204,7 +204,7 @@ func TestProposer_GetBlock_AddsUnaggregatedAtts(t *testing.T) {
 	assert.Equal(t, false, hasUnaggregatedAtt, "Expected block to not have unaggregated attestation")
 }
 
-func TestProposer_ProposeBlock_OK(t *testing.T) {
+func TestProposer_ProposeBlock_Phase0_OK(t *testing.T) {
```

Comment:
My personal opinion is that `t.Run` is better than separate tests because it allows for common setup. But either way is fine.